### ISSUE TITLE
Latest binary has broken walking

### ISSUE THREAD

**Issue by MaciekBaron** [teamsalta/sm64nx#26]
It seems that the latest binary in the repo has broken walking, when using an XBox controller.

[The first version of the binary](https://github.com/teamsalta/sm64nx/blob/d591a80a520d2a13be7563de0d4139a61d42f0fa/sm64.exe) seems to be working fine. In the latest binary, Mario seems to be moonwalking/stuck in place (he responds to the controller though).

**Comment by RinMaru** [comment]
yea upon first boot he didnt respond to the analog but after going into controller settings he all the sudden moves again

**Comment by MaciekBaron** [comment]
You're right, if you go into controller settings everything suddenly works correctly, and then upon rerunning the executable.

**Comment by RinMaru** [comment]
would be great if they added in icons or texts saying what everything is mapped to
